frontend/server/src/Controllers/Problem.php

```diff
@@ -3772,7 +3774,8 @@ public static function apiList(\OmegaUp\Request $r) {
             $r->identity,
             $r->user,
             $onlyQualitySeal,
-            $level
+            $level,
+            $difficulty ?: 'all'
```

Comment:
```suggestion
            $difficulty ?? 'all'
```

por consistencia con L5925

Reply:
Al ponerlo así me mercaba un error, ¿es correcto modificar L5925 con `?:`?

Reply:
qué error te sale? preferiría tener `??` si es posible.